docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/domain-events-design-implementation.md

```diff
@@ -267,51 +266,56 @@ Based on the relationship between event and event handler, which can be consider
 
 ### How to handle domain events
```

Comment:
there's something odd with the layout here since GitHub is still interpreting this as code

Reply:
I don't see that wrong..